### ISSUE TITLE
[FEATURE] Add export/import functionality to single types

### ISSUE THREAD

**Issue by bass3l** [Baboo7/strapi-plugin-import-export-entries#25]
First, thank you for this cool plugin. 👍 

**Is your feature request related to a problem? Please describe.**
It's not related to a problem. Currently, I'm building a website where the index page (which should always be present) is being modeled as a single type. This particular type has a lot of components and dynamic zones that can be used to define your index page.

**Describe the solution you'd like**
I think it's a valid use-case to be able to export/import a single type just like we do with the collections. A simply export and import buttons can be at the top beside the single type 'save' button, where an import simply overwrites all the single type data.

**Additional context**
I would really like to help out developing this feature. If you think this is a valid feature to be added, give me the go and I'll arrange a pull request where we can discuss it further. Thanks.

**Comment by Baboo7** [comment]
Good point. I added it to the task queue

**Comment by Baboo7** [comment]
Version `1.8.0` out. Single types are can now be imported/exported!

**Comment by shawnvogt** [comment]
Thank you @Baboo7 for this project. 

I was looking for this functionality as well and decided to take a swing at it myself, but I really just did the absolute minimum to get this working well enough for the project I am working on's needs. It is also not likely the best approach (I only have a few hours looking through the codebase, and I am new to Strapi). 

I just wanted to share in case there is anything potentially useful in the code I added. My changes can be viewed [here](https://github.com/TMRRWinc/strapi-plugin-import-export-entries/commit/d4a4c8488b3de305c7334ecbc342112bf7c0b227) 

The approach I took was to inject the import and export buttons into the edit view right links and hide them when editing a single element of a collection type. Other than that, the entries are converted to an array in the export service if entries are not already an array so it plays nice with the existing export functionality.  

And as I am typing this, I see that you released version 1.8.0 with Single Type functionality. Lol, I guess I should have looked for this functionality today instead of yesterday. 

Thank you for your time on this project.
